check_pep8.py

```diff
@@ -1,5 +1,35 @@
-#!/usr/bin/env python
-import subprocess
+import os
+import unittest
 
-subprocess.call(['flake8', '--exclude',
-                 'env, dreamjub/migrations, login/migrations', '.'])
+import pep8
+
+ignore_patterns = ('env', '.git', '__pycache__', 'dreamjub/migrations')
```

Comment:
Why only exclude migrations from one app? What about `login`, `portal` and alike?